src/js/helpers/nodes/nodesHelper.js

```diff
@@ -13,6 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import { DataSet } from 'vis-timeline';
```

Comment:
I would add blank line to be consist with the rest of the code base.

Reply:
Not sure what you mean? Do you mean empty line after this import or spreading this import into multiple lines? Because I think neither of these would be consistent

Reply:
Adding a blank line before that import. At least this is what I've seen you usually do to separate licence and the rest of the code.